deploy/conf/php-fpm.conf

```diff
@@ -5,7 +5,7 @@ listen = /var/run/php-fpm.sock
 listen.owner = www-data
 listen.group = www-data
 listen.mode = 0660
-pm = dynamic
+pm = static
 pm.max_children = 5
```

Comment:
The change to static is really good.
But why only 5 children, we know the server machine specs ??